cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -36,28 +36,40 @@ class RightNav{
         return this.testDocumentWithTitle(tab, subTab, title, 'not.contain');
     }
 
-    getRightNavTabs(){
+    getPrimaryWorkspaceTabs(){
```

Comment:
All tabs are now in the resource panel and should probably be consolidated there instead of here.